### ISSUE TITLE
Component is not accessible to keyboard users

### ISSUE THREAD

**Issue by togakangaroo** [localvore-today/react-redux-mapbox-autocomplete#18]
When typing an address into the component I get a popup as I'm supposed to. However, there seems to be no way to navigate to the popup with the keyboard! Selections also don't show up as areas you can take action on.

This is a big problem for keyboard bound users or ones who have difficulty using mouse/touch (eg individuals with MS, Parkinsons, or just using keyboard/speech assist tools)
